--- a/setup.py
+++ b/setup.py
@@ -119,11 +119,6 @@ setup(
             "transformers>=5.9.0"
         ),
         ("datasets>=4.8.4,<=5.0.1" if BUILD_TYPE == "release" else "datasets>=4.8.4"),
-        (
-            "auto-round>=0.14.1,<=0.14.2"
-            if BUILD_TYPE == "release"
-            else "auto-round>=0.14.1"
-        ),
         (
             "accelerate>=1.6.0,<=1.14.0"
             if BUILD_TYPE == "release"
@@ -142,6 +137,13 @@ setup(
         ),
     ],
     extras_require={
+        "autoround": [
+            (
+                "auto-round>=0.14.1,<=0.14.2"
+                if BUILD_TYPE == "release"
+                else "auto-round>=0.14.1"
+            ),
+        ],
         "dev": [
             # testing framework
             "pytest>=6.0.0",
